Mixpanel/Mixpanel.h

```diff
@@ -96,7 +96,7 @@ NS_ASSUME_NONNULL_BEGIN
  @discussion
  Setting a flush interval of 0 will turn off the flush timer.
  */
-@property (atomic) NSUInteger flushInterval;
+@property (nonatomic) NSUInteger flushInterval;
```

Comment:
The `atomic` / `nonatomic` properties don't change anything for us since we've overridden the setter and we're not using the synthesized code.

I'll change to `atomic` and add `@synchronize` to ensure the public API requirement is met.